Assets/EngyneCreations/Trello/Scripts/TrelloBoardSettings.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Trello
+{
+	public class TrelloBoardSettings : ScriptableObject
+	{
+		[Header("API Key and Token needed for Interacting with Trello")]
+		public string APIKey;
+		public string APIToken;
+
+		private static TrelloBoardSettings _instance;
+
+		public static TrelloBoardSettings Instance
+		{
+			get
+			{
+				LoadSettings();
+				return _instance;
+			}
+		}
+
+		public static void LoadSettings()
+		{
+			_instance = FindOrCreateInstance();
+		}
+
+		private static TrelloBoardSettings FindOrCreateInstance()
+		{
+			TrelloBoardSettings instance = null;
+			instance = Resources.Load<TrelloBoardSettings>("TrelloBoard");
+			instance = instance ? instance : Resources.LoadAll<TrelloBoardSettings>(string.Empty).FirstOrDefault();
+			instance = instance ? instance : CreateAndSave<TrelloBoardSettings>();
```

Comment:
instead of the stacking and repetition of `instance`, 
is it easier to read
`Resources.Load` ?? `ResourcesLoadAll` ?? `CreateAndSave`

Reply:
if you have
`Resources.LoadAll<TrelloBoardSettings>(string.Empty).FirstOrDefault`
i don't think you need the first `Resources.Load` since it covers it.

Reply:
Realized that `Resources.LoadAll` shouldn't be used at all, since its slower and pre-supposing the settings file is not the given location when I have given only one location for the file to be in. 

Using `Resources.Load` instead